package.js

```diff
@@ -54,7 +54,7 @@ Package.onTest(function (api) {
     'peerlibrary:assert@0.2.5',
     'peerlibrary:server-autorun@0.7.1',
     'peerlibrary:classy-test@0.3.0',
-    'meteorhacks:unblock@1.1.0'
+    'lamhieu:unblock@1.0.0'
```

Comment:
What is this change?

Reply:
`meteorhacks:meteorx` which is used by `meteorhacks:unblock` doesn't work with Meteor 1.8.1. I've been using `lamhieu:meteorx` and `lamhieu:unblock` with Monti APM with no problems so far.